colibre/description.html

```diff
@@ -99,8 +99,19 @@ <h4>Stellar Feedback</h4>
         <td>\(v_{\rm kinetic}\)</td>
         <td>{{ "%.4g" % (data.metadata.parameters["COLIBREFeedback:SNII_delta_v_km_p_s"] | float) }} km s\(^{-1}\)</td>
     </tr>
+     <tr>
+        <td>\(\Delta T\)</td>
+        <td>{{ data.metadata.parameters["COLIBREFeedback:SNII_delta_T_K"] | float | format_number }} K</td>
```

Comment:
Please use `.get("COLIBREFeedback:SNII_delta_T_K", default_value)` for things that may not be present in everybody's run.

Reply:
That is in principle true for all stellar-feedback related fields. Shall I pick `0.0` as the default value for all of them?

Reply:
Yes, that would be best, especially at this stage where we might add/remove fields.

Reply:
Alright, 

Also, I've been thinking, sometimes it might be unclear if a parameter is not specified or explicitly set to zero. Maybe in the former case we could write `None` or something like that.

Reply:
Indeed but this is more complex when you pass everything through the float and format_number thing... I could write a function called get_if_present_float and get_if_present_int or something? I think the best idea if not present is just to have an empty string.

Reply:
Yes, that'd be great.

Sure, let's use an empty string.